lib/zht/pages/ActualPosition.dart

```diff
@@ -38,27 +40,42 @@ class ActualPositionState extends State<ActualPosition>{
     }
     
     _locationData = await location.getLocation();
```

Comment:
Ok, you can stop to used the setState. But, why don't you user the provider instead of a local variable to save this data ?